build.rs

```diff
@@ -0,0 +1,21 @@
+use std::process::Command;
+
+fn main() {
```

Comment:
What's this for? :thinking:

Reply:
Setting a build-time `GIT_HEAD` env var, so that we can include it in the logs when starting the server.

Reply:
Cool, thanks for explaining :)